webpack.config.js

```diff
@@ -2,11 +2,14 @@ var path = require('path');
 
 module.exports = {
   entry: './src/automerge.js',
+  mode: 'development',
   output: {
     filename: 'automerge.js',
     library: 'Automerge',
     libraryTarget: 'umd',
-    path: path.resolve(__dirname, 'dist')
+    path: path.resolve(__dirname, 'dist'),
+    // https://github.com/webpack/webpack/issues/6525
+    globalObject: 'this'
```

Comment:
@salzhrani Just to understand what's going on, could you explain what this `globalObject` declaration means? Is the intention that we change it when webpack/webpack#6525 is implemented?